package.json

```diff
@@ -16,23 +16,24 @@
     "registry": "https://registry.npmjs.org"
   },
   "dependencies": {
-    "classnames": "^2.2.5",
     "@types/classnames": "^2.2.3",
+    "@types/lodash": "^4.14.74",
+    "@types/node": "^12.6.2",
```

Comment:
We should use version 10 to match the node version https://github.com/Clever/components/blob/b91d962eff2e6d9b4028e52bcce8e4dc157d3f2e/Makefile#L4

Reply:
3c101ec